packages/experimental/Expander/windows/ReactNativeExpander/ReactNativeExpander.vcxproj

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looks like this project might have been created with an older version of RNW which doesn't have ExperimentalFeatures.props - this is important in order to enable apps using this module to customize their own version of the SDK, WinUI, etc.

Reply:
I created the project with 0.63.0 initially since the FURN repo required it. Is there a good way to allow for customization without generating a new project? Or would generating a 0.64.0 project and copying the important files over be the easiest?